### PR TITLE
Fix height bug in loginpage and refactor to vanilla css

### PR DESCRIPTION
**Pull request by gustaveen**
#### Before
Screenshot 2024-05-22 at 10 07 23

#### After
Screenshot 2024-05-22 at 10 07 30

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
* **#4298** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4298?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @gustaveen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/features/memberArea/pages/LoginPage/LoginPage.css.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { style } from '@vanilla-extract/css'
+import { minWidth, theme } from 'ui'
```
Comment (Youakeem) [inline apps/store/src/features/memberArea/pages/LoginPage/LoginPage.css.ts:2]: Let's use `tokens` instead of `theme` to use the css variables.

Comment (gustaveen) [reply apps/store/src/features/memberArea/pages/LoginPage/LoginPage.css.ts:2]: Aha, didn't know we had changed to that!

**Review by Youakeem — APPROVED** [review]

**Comment by gustaveen** [comment]
### Merge activity

* **May 23, 5:18 AM EDT**: @gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4298).

**Pull request merged**